p2p/net/swarm/swarm.go

```diff
@@ -602,6 +603,38 @@ func isBetterConn(a, b *Conn) bool {
 	return true
 }
 
+func connPriority(c *Conn) int {
+	if c == nil {
+		return 0
+	}
+
+	var priority int
+	// LAN > WAN > PROXY
+	switch {
+	case c.conn.Transport().Proxy():
+		priority += 10
+	case manet.IsPrivateAddr(c.RemoteMultiaddr()):
+		priority += 20
+	case manet.IsPublicAddr(c.RemoteMultiaddr()):
+		priority += 15
+	}
+
+	// We prefer udp protocols
+	switch c.ConnState().Transport {
+	case "quic", "quic-v1", "webtransport", "webrtc":
+		priority += 5
+	case "tcp", "websocket":
+		priority += 3
+	}
+
+	//  If one is transient and not the other, prefer the non-transient connection.
+	if !c.Stat().Transient {
+		priority += 2
+	}
+
```

Comment:
```suggestion
```

This now do nothing because both connections have identical transience.

Reply:
Sorry, I must be an idiot, I forgot to delete this part of the code...

Reply:
<3 you're not